### DISCUSSION TITLE
Welcome to xray-junit-extensions Discussions!

### DISCUSSION THREAD

**Discussion by bitcoder** [Xray-App/xray-junit-extensions#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kprasad-medibank** [comment]
Hello, community
Hi, I’m Krishna, an Android engineer. I’d like to find out if this extension can be used with Android apps to generate customized test reports in the build folder.

I’ve added the dependencies to my build.gradle and can access the library. However, I’m a bit unsure about how to register EnhancedLegacyXmlReportGeneratingListener in the code. Could someone shed some light on this?
